### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ZhuchkaTriplesix**
Add endpoint and logic for transactions, order, service orders

### CODE REVIEW

**Review by ZhuchkaTriplesix — COMMENTED** [review]
Thanking good, wait for ph4

File: api/main.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def employee_delete(id: int):
     if emp is False:
         raise HTTPException(status_code=404)
     else:
-        return True
+        raise HTTPException(status_code=200)
```
Comment (ph4) [inline api/main.py:82]: Fastapi returns success by default, no point in raising you can remove else branch.
Best to use 204 (No Content) if no response body is sent.
`@app.delete("/employees/{id}", tags=["Employee"], status_code=204)`

File: api/functions.py [hunk 2]
```diff
@@ -682,12 +765,27 @@ def update(id: int, name: str, deliver_service: str) -> DistributorDantic:
...
-    def add(component_id: int, count: float, distributor: int):
+    def add(component_id: int, count: float, distributor_id: int) -> OutputSupplyDantic:
         sess = Session()
-        supply = Supplies(component_id=component_id, count=count, distributor=distributor)
-        sess.add(supply)
-        sess.commit()
-        sess.close()
+        try:
+            supply = Supplies(component_id=component_id, count=count, distributor=distributor_id)
+            print(1)
```
Comment (ph4) [inline api/functions.py:772]: Check out https://github.com/gruns/icecream 🍧
It's very cool and you can disable debug output.

File: docker-compose.yaml [hunk 3]
```diff
@@ -1,6 +1,14 @@
 version: '3.5'
 
 services:
+  api:
+    build: .
+    container_name: fastapi-application
+    ports:
+      - '8000:8000'
+    command: pip install uvicorn
```
Comment (ph4) [inline docker-compose.yaml:9]: Aren't requirements installed in Dockerfile?

Comment (ZhuchkaTriplesix) [reply docker-compose.yaml:9]: yes

**Review by ZhuchkaTriplesix — COMMENTED** [review]
<3

**Pull request merged**